accounts-db/src/active_stats.rs

```diff
@@ -17,6 +27,16 @@ pub struct ActiveStats {
 #[derive(Debug, Copy, Clone)]
 pub enum ActiveStatItem {
     Clean,
+    CleanCollectCandidates,
```

Comment:
wow. this is a lot of granularity. my gut says we need half of these. otoh, what is the downside of all these? some noise? not sure. certainly helpful as we debug these oom situations. not sure how useful it is usually. since they are metrics it also seems that we don't need log files to find 'info' logs. I'm thinking we don't really need the 1/0 results. maybe if we just logged the 1 each time each phase started and we could look at the code to see there are no gaps between stages. that seems kind of hacky and just trying to save half the logs. each of these logs will be once per clean, so once per 100 slots. This is a rambling comment. In short, I'm reluctant to put in this much granularity. But, maybe it is hte right idea and I shouldn't be so conservative?